cmd/gen/dependabot/command.go

```diff
@@ -12,9 +12,9 @@ import (
 const (
 	name        = "dependabot"
 	description = "Generates GitHub Dependabot config for go and docker dependencies (.github/dependabot.yml)."
-	example     = `  devctl gen dependabot 
-  devctl gen dependabot --interval daily --reviewers giantswarm/team-firecracker
-  devctl gen dependabot --interval weekly --reviewers giantswarm/team-firecracker,njuettner`
+	example     = `  devctl gen dependabot --ecosystems go
+  devctl gen dependabot --ecosystems docker,go --interval daily --reviewers giantswarm/team-firecracker
+  devctl gen dependabot --ecosystems docker --interval weekly --reviewers giantswarm/team-firecracker,njuettner`
```

Comment:
I'm ok with that if you have a strong preferenc but maybe it would be simpler for the end user if the code would simply check `Dockerfile` existence?

Reply:
So you would drop out `Docker` as an exception but other package ecosystems would be ok like GH actions, terraform or npm? 

Would be ok to me I guess, I was not sure if we always use 'Dockerfile' as a file name or if something else might be also is used. I need some confirmation on that.

Reply:
To that point, it might be possible to make this optional and detect a Dockerfile / go.mod / yarn.lock / etc. and allow this to override the auto-detection

Reply:
If the repo is not using a standard name for these dependency files, then Dependabot can't find it anyway and the repo would have to apply a manual setting in the Dependabot config

Reply:
Ok sounds good, will change it a bit for the autodetection. Thanks both of you for the input 👍

Reply:
I couldn't find a list of the specific filenames they look for (they're spread between ruby modules) but here's a list of the languages at least https://dependabot.com/#languages

Reply:
Let's maybe not go too crazy and detect only what we use and extend slowly.